flask_menu/ext.py

```diff
@@ -44,6 +47,10 @@ def url_preprocessor(route, args):
             g._menu_kwargs = args
             g._menu_route = route
 
+    def init_config(self, app):
+        "Initialize configuration."
+        app.config["FLASK_MENU"] = config.FLASK_MENU
```

Comment:
minor: to follow the convention in other modules
```suggestion
        app.config.setdefault("FLASK_MENU", getattr(config, "FLASK_MENU"))
```